inspirehep/modules/workflows/utils.py

```diff
@@ -176,3 +177,41 @@ def download_file_to_workflow(workflow, name, url):
         if req.status_code == 200:
             workflow.files[name] = req.raw
             return workflow.files[name]
+
+
+def store_original_record_for_source(record_id, source, data):
```

Comment:
Yup, usage soon to come in additional commits.

Possible better name could be:
* `store(/put/persist?)_root_json()`
* `retrieve(/get/recover?)_root_json()`

WDYT?